homeassistant/components/samsungtv/config_flow.py

```diff
@@ -96,81 +151,90 @@ async def async_step_import(self, user_input=None):
     async def async_step_user(self, user_input=None):
         """Handle a flow initialized by the user."""
         if user_input is not None:
-            ip_address = await self.hass.async_add_executor_job(
-                _get_ip, user_input[CONF_HOST]
-            )
-
```

Comment:
Looking it up is likely fine here since we want to store the ip

Reply:
Readded